frontend/asset_mapper.rst

```diff
@@ -0,0 +1,1030 @@
+
+In the past, it was common to combine assets to reduce the number of HTTP
+requests that were made. Thanks to advances in web servers like
+HTTP/2, it's typically not a problem to keep your assets separate and let the
+browser download them in parallel. In fact, by keeping them separate, when
+you update one asset, the browser can continue to use the cached version of
+all of your other assets.
+
+See :ref:`Optimization <optimization>` for more details.
+
+Does AssetMapper Minify Assets?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Nope! Minifying or compressing assets *is* important, but can easily be
+done by your web server or a service like Cloudflare. See
+:ref:`Optimization <optimization>` for more details.
+
+Is AssetMapper Production Ready? Is it Performant?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes! Very! AssetMapper leverages advances in browser technology (like importmaps
+and native ``import`` support) and web servers (like HTTP/2, which allows assets
+to be downloaded in parallel). See the other questions about minimization and
+combination and :ref:`Optimization <optimization>` for more details.
+
+The https://ux.symfony.com site runs on AssetMapper and has a 99%
+Google Lighthouse score.
+
+Does AssetMapper work in All Browser?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yup! Features like importmaps and the ``import`` statement are supported
+in all modern browsers, but AssetMapper ships with an `es module shim`_
+to support ``importmap`` in old browsers. So, it works everywhere (see note
+below).
+
+Inside your own code, if you're relying on modern `ES6`_ JavaScript features
+like the `class syntax`_, this is supported in all but the oldest browsers.
+If you *do* need to support very old browsers, you should use a tool like
+:ref:`Encore <frontend-webpack-encore>` instead of AssetMapper.
+
+.. note::
+
+    The `import statement`_ can't be polyfilled or shimmed to work on *every*
+    browser. However, only the **oldest** browsers don't support it - basically
+    IE 11 (which is no longer supported by Microsoft and has less than .4%
+    of global usage).
+
+    The ``importmap`` feature **is** shimmed to work in **all** browsers by AssetMapper.
+    However, the shim doesn't work with "dynamic" imports:
+
+    .. code-block:: javascript
+
+        // this works
+        import { add } from './math.js';
+
+        // this will not work in the oldest browsers
+        import('./math.js').then(({ add }) => {
+            // ...
+        });
+
+    If you want to use dynamic imports and need to support certain older browsers
+    (https://caniuse.com/import-maps), you can use an ``importShim()`` function
+    from the shim: https://www.npmjs.com/package/es-module-shims#user-content-polyfill-edge-case-dynamic-import
+
+Can I Use with Sass or Tailwind?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sure! See :ref:`Using Tailwind CSS <asset-mapper-tailwind>` or :ref:`Using Sass <asset-mapper-sass>`.
```

Comment:
Might be useful to add a little sentence that it requires a separate build step?